marrow/mailer/transport/ses.py

```diff
@@ -5,55 +5,55 @@
     from botocore.exceptions import ClientError
 
 except ImportError:
-    raise ImportError("You must install the boto package to deliver mail via Amazon SES.")
+    raise ImportError(
+        "You must install the boto package to deliver mail via Amazon SES.")
 
 
 __all__ = ['AmazonTransport']
 
 log = __import__('logging').getLogger(__name__)
 
 
-
-class AmazonTransport(object): # pragma: no cover
+class AmazonTransport(object):  # pragma: no cover
     __slots__ = ('ephemeral', 'config', 'region', 'connection')
-    
+
     def __init__(self, config):
         # Give our configuration aliases their proper names.
         config['aws_access_key_id'] = config.pop('id')
         config['aws_secret_access_key'] = config.pop('key')
-        
+
         self.region = config.pop('region', "us-east-1")
-        config.pop('use') #boto throws an error if we leave this in the next line
+        # boto throws an error if we leave this in the next line
+        config.pop('use')
         config.pop('debug')
-        self.config = config  # All other configuration directives are passed to connect_to_region.
+        # All other configuration directives are passed to connect_to_region.
+        self.config = config
         self.connection = None
-    
+
     def startup(self):
-        self.connection = boto3.client('ses',region_name=self.region, **self.config)
-    
+        self.connection = boto3.client(
+            'ses', region_name=self.region, **self.config)
+
```

Comment:
120-column readability example here. The original was fine as a single line, it was the `'ses',region_name` lack of whitespace between the arguments that was the concern.

Reply:
Agree, changed.